doc/CMakeLists.txt

```
@@ -1,3 +1,53 @@
+set(VENVDIR ${CMAKE_CURRENT_BINARY_DIR}/sphinx/venv)
+set(BUILDDIR ${CMAKE_CURRENT_BINARY_DIR}/sphinx/_build)
+set(MAKEOPTS
+  VENVDIR=${VENVDIR}
+  BUILDDIR=${BUILDDIR}
+  DOCTREESDIR=${BUILDDIR}/../.doctrees
+)
+set(SPHINXDIR ${CMAKE_CURRENT_SOURCE_DIR}/sphinx)
+
+add_custom_command(
+  COMMAND make -C ${SPHINXDIR} install
+          VENVDIR=${VENVDIR}
+  OUTPUT ${VENVDIR}
+)
+
+set(SPHINX_TARGETS
+  clean
+  clean-doc
+  html
+  linkcheck
+  lint-md
+  pa11y
+  run
+  serve
+  spelling
+  vale
+  woke
+)
+
+foreach(target IN LISTS SPHINX_TARGETS)
+  add_custom_target(doc-${target}
+    # In `VAR1=<value> make <target> VAR2=<value>, only `VAR1` is inherited by
+    # `make`'s children, `VAR2` only applies within `make`.
+    COMMAND ${CMAKE_COMMAND} -E env "MIR_CMAKE_BUILD_DIR=${CMAKE_BINARY_DIR}"
```

Didn't I rename this?…
```suggestion
    COMMAND ${CMAKE_COMMAND} -E env "MIR_BUILD_DIR=${CMAKE_BINARY_DIR}"
```